### ISSUE TITLE
Only got two spies from Order's Double Agents

### ISSUE THREAD

**Issue by pandasnail** [LoneGazebo/Community-Patch-DLL#3743]
_1. Mod version (i.e Date - 4/23):_Dec 6 hotfix

_2. Mod list (if using Vox Populi only, leave blank):_
Full EUI VP
Randomize City Names, Celtic City Names, Iroquian City Names
Civ Linked Great Generals, Great People Pack. Great Prophet Historical Names, More Great Musicians, More Great Works
Favorite Religion Randomizer, Historical Religions
Flag Promotions
My own thing, which these days only removes the barb xp cap and renames some City States

_3. Error description:_ 
Double Agents says it gives three, and the code even does says FreeSpy = '3', but nonetheless I got only two. Fwiw, playing with 28 City States, two was also my normal Era advance allotment.

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by bigcat88** [comment]
It depends from number of city states or map size, don't remember. Cause in one my game I got 3 spies, and in other 2...

**Comment by pandasnail** [comment]
Yeah, the Freedom one and the Statecraft policy work from City States, which I think is always just one tenth rounded down, but Double Agents says it'll be exactly three. If its suppose to work those others, then this is just a description thing.

**Comment by TechpriestEnginseer** [comment]
They shouldn't be since FreeSpy is used by the same Statecraft and Freedom respective policy.

**Comment by LoneGazebo** [comment]
With that many CSs it is possible that you ran into the max # of spies from your spy list.

**Comment by pandasnail** [comment]
??? 28 is not that many city states, its still only two spies per era.

But more importantly, and I realize since I didn't post a screenshot you couldn't have known this, I wanted allllllll the Great Works, so I took it early. I can't remember now if it was Industrial or Modern when I took it, but I for sure didn't have the Atomic Spies, the Information spies, NIA, etc. And I didn't go Statecraft either. I don't think its that.
